packages/perseus/src/widgets/interactive-graphs/utils.ts

```diff
@@ -1,3 +1,5 @@
+import * as SimpleMarkdown from "@khanacademy/pure-markdown";
```

Comment:
With this suggestion, line 86 would be changed to 
`const firstSection = parse(mathString)[0];`

Reply:
It's true that that's the more streamlined way to do it, but I have a personal preference to keep it as `SimpleMarkdown.parse` because I feel that it makes it more discoverable in the IDE search. There are a bunch of different parse functions, so I like being very explicit about which package this one is coming from.